kibana/utils

```diff
@@ -85,7 +85,7 @@ update_config_from_env_vars() {
         if [ -n "${DEBUG:-}" ] ; then
           echo "updating ${conf_file} - '${kibana_var}: ${value}'"
         fi
-        echo "${kibana_var}: ${value}" >> ${conf_file}
+        sed -i "s,${kibana_var}:.*,${kibana_var}:\ ${value}," "${conf_file}"
```

Comment:
This only will work if the supported var is actually defined in the config file.  What if it is not?

Reply:
Good catch we need to add this logic too. However it would never have worked with the old append, because it destroy the very last config option if any available or amend a comment line.